admin/app/components/solidus_admin/base_component.rb

```diff
@@ -38,5 +38,10 @@ def spree
     def solidus_admin
       @solidus_admin ||= SolidusAdmin::Engine.routes.url_helpers
     end
+
+    def self.inherited(child)
+      super
+      child.include SolidusAdmin::Engine.routes.url_helpers
+    end
```

Comment:
I think I would prefer a method `routes`, so that we can a) extend it and b) easily separate the routes from regular methods in the components.